### ISSUE TITLE
Comment Typo in asNSImage

### ISSUE THREAD

**Issue by jhokit** [dagronf/SwiftImageReadWrite#1]
func asNSImage ends with this comment, which should say NSImage

/// Create a UIImage representation of this image

**Comment by dagronf** [comment]
@jhokit I can't find this issue, but the `asPlatformImage()` for macOS had this issue.

Is this the one you mean?

**Comment by jhokit** [comment]
Sorry Darren, I got it wrong. Yes, in asPlatformImage().

- Jeff

**Comment by jhokit** [comment]
I was reading lots and lots of image code today and my head was spinning. 😵‍💫
Yours is very clear, good stuff.

**Comment by dagronf** [comment]
@jhokit I know what you mean 😀.  Having CGImage, NSImage, UIImage, SwiftUI.Image formats to support you're writing cross-platform code is kind of a nightmare. 

I kept finding myself dropping down to using CGImage for everything, but then I had to cope with all the things that NSImage/UIImage provides for free. Hence why I started writing this library!

Glad it's been of help to you!  I'll push up a new version soon with your fix.
